README.md

```diff
@@ -1,49 +1,97 @@
+- **Tooling Improvements**: Developing new or improving existing tools for modding and game development.
+- **Community-Driven Improvements**: Once the community grows, we plan to incorporate more features, updates, and
+  changes based on player feedback.
 
-The quickest way to build all configurations in the project is to open `rts.dsw` in Microsoft Visual Studio C++ 6.0 (SP6 recommended for binary matching to Generals patch 1.08 and Zero Hour patch 1.04) and select Build -> Batch Build, then hit the “Rebuild All” button.
+## Running the Game
 
-If you wish to compile the code under a modern version of Microsoft Visual Studio, you can convert the legacy project file to a modern MSVC solution by opening `rts.dsw` in Microsoft Visual Studio .NET 2003, and then opening the newly created project and solution file in MSVC 2015 or newer.
+To run *Zero Hour* using this project, you need to have the original *Command & Conquer: Generals and Zero Hour* game
+installed. The easiest way to get it is through *Command & Conquer The Ultimate Collection*
+on [Steam](https://store.steampowered.com/bundle/39394). Once the game is ready, download the latest version of the
+project from [GitHub Releases](https://github.com/TheSuperHackers/GeneralsGameCode/releases), extract the necessary 
+files, and follow the instructions in the [Wiki](https://github.com/TheSuperHackers/GeneralsGameCode/wiki).
 
-NOTE: As modern versions of MSVC enforce newer revisions of the C++ standard, you will need to make extensive changes to the codebase before it successfully compiles, even more so if you plan on compiling for the Win64 platform.
+### Disclaimer
 
-When the workspace has finished building, the compiled binaries will be copied to the folder called `/Run/` found in the root of each games directory. 
+The current builds may have bugs or unexpected behavior. We will aim to release stable versions that are as free from
+bugs as possible, but even those may have some unexpected behaviors.
 
+### Antivirus Warning
 
-## Known Issues
+Some antivirus software may mistakenly flag this project as a false positive (e.g., Windows Defender). We assure you
+that the project is fully safe to use. However, if you have concerns, you can
+always [build it yourself](#building-the-game-yourself).
 
-Windows has a policy where executables that contain words “version”, “update” or “install” in their filename will require UAC Elevation to run. This will affect “versionUpdate” and “buildVersionUpdate” projects from running as post-build events. Renaming the output binary name for these projects to not include these words should resolve the issue for you.
+## Joining the Community
 
+You can chat and discuss the development of the project on our
+[Discord channel](https://www.community-outpost.com/discord) or on 
+our [GameReplays forum](https://www.gamereplays.org/community/index.php?showforum=132). Join us to get the latest
```

Comment:
I suggest to remove the link to GameReplays. It is not relevant for this project.